CHANGELOG.rst

```diff
@@ -112,6 +112,8 @@ Added
 - Add a `bot_challenge` intent and a `utter_iamabot` action to all example projects and the rasa init bot.
 - Allow sending attachments when using the socketio channel
 - ``rasa data validate`` will fail with a non-zero exit code if validation fails
+- `CRFEntityExtractor` updated to accept arbitrary token-level features like word vectors (issues/4214)
```

Comment:
This ended up in the wrong section. Can you move it to the top of the changelog file into section 1.4.x? Thanks.